packages/wallets/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@lidofinance/wallets-testing-wallets",
+  "version": "0.0.0-semantic-release",
```

Comment:
Should it be 0.0.0?

Reply:
yep, it's just placeholder and actual version will be taken from git tags